### PR TITLE
Update mcr.cpp

### PR DESCRIPTION
**Pull request by AuthurXi**
tips for player should be changed from "Which cellto mark? i:[1..3], j:[1..3]:"to"Which cellto mark? i:[0..2], j:[0..2]:"

### CODE REVIEW

**Review by FacundoChan — COMMENTED** [review]
There is a problem with using i==3 to determine a tie. Instead, use n==9 to check for a tie, where n is defined as a global variable.

**Review by coolcheng222 — COMMENTED** [review]
Good update, and you should check wheathe ij is within 0 and 2

**Review by Mtlala — COMMENTED** [review]
Need empty line to sperate different blocks of codes to enhance its readability. 
And it need brackets after if and else (though it still works in some cases).

File: mcr.cpp [hunk 1]
```diff
@@ -26,7 +26,7 @@ int main(){
 				cout << "Player 1: ";
 			else
```
Comment (Mtlala) [inline mcr.cpp]: Need brackets after if and else (though it still works in some cases).

File: mcr.cpp [hunk 2]
```diff
@@ -26,7 +26,7 @@ int main(){
...
 				cout << "Player 2: ";
-			cout << "Which cell to mark? i:[1..3], j:[1..3]: ";
```
Comment (Mtlala) [inline mcr.cpp:29]: Need empty line to sperate different blocks of codes to enhance its readability.

**Comment by Mtlala** [comment]
你好 我是陈彦彤~邮件已经收到啦~我会尽快回复~

File: mcr.cpp (hunk 1 above)
Comment (AuthurXi) [reply mcr.cpp]: ok,it will be fixed

File: mcr.cpp (hunk 2 above)
Comment (AuthurXi) [reply mcr.cpp:29]: ok it will be fixed

File: mcr.cpp (hunk 1 above)
Comment (Mtlala) [reply mcr.cpp]: Approved

File: mcr.cpp (hunk 2 above)
Comment (Mtlala) [reply mcr.cpp:29]: Approved

**Review by Mtlala — APPROVED** [review]

**Pull request merged**